robottelo/performance/candlepin.py

```diff
@@ -121,7 +121,7 @@ def single_delete(cls, uuid, thread_id):
             LOGGER.error(
                 'Fail to delete {0} on thread-{1}!'.format(uuid, thread_id))
             LOGGER.error(response.content)
-            return
+            return 0
```

Comment:
yep..the point is for generating reports or csv statistics: if set as None, numpy won't accept a list of None and thus throw error; else if set as 0, the script would continue to end.